README.md

```diff
@@ -43,7 +43,7 @@ The action works on these [GitHub-hosted runners](https://docs.github.com/en/act
 | Operating System | Supported |
 | ---------------- | --------- |
 | Ubuntu  | `ubuntu-20.04`, `ubuntu-22.04` |
-| macOS   | `macos-11`, `macos-12`, `macos-13`, `macos-13-xlarge` (arm64) |
+| macOS   | `macos-11`, `macos-12`, `macos-13`, `macos-13-xlarge` (arm64), `macos-14` (arm64) |
```

Comment:
And shouldn't this include `macos-14-large` and `macos-14-xlarge`? And also `macos-13-large`?

Are we sure `macos-14` is running on ARM? I can see that GitHub writes two different things about the runners.
Here they write that `Large` is Intel, `XLarge` is arm64:
https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#about-macos-larger-runners

Here they write that `macos-14` is M1:
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
```suggestion
| macOS   | `macos-11`, `macos-12`, `macos-13`, `macos-13-large`, `macos-13-xlarge` (arm64), `macos-14`, `macos-14-large` (arm64), `macos-xlarge` (arm64) |
```

Reply:
Please read the description of this PR again and link referenced there.

Reply:
Also, I think `macos-13-xlarge` got a special callout only because it was the first and the only github hosted macos arm64 runner at the time. Typically larger runners are not explicitly listed in the doc.

Reply:
And finally, M1 is arm64. https://en.wikipedia.org/wiki/Apple_M1

Reply:
Yes you're right, I misunderstood the first link after reading the next ones. And if `macos-13-large` was a "special" one, please ignore my comments!

(but `macos-13-xlarge` should probably be removed now that `macos-14` is out, either right away or at a later point)

Reply:
Updated README to remove `macos-13-xlarge`.